hooks/local-monitors-relation-changed.ps1

```diff
@@ -41,9 +39,6 @@ function Set-LocalMonitors {
                     'hyper_v_virtual_switch_packets_per_sec' = @{
                         'command' = "CheckCounter -a Counter=`"\\Hyper-V Virtual Switch($switchName)\\Packets/sec`""
                     };
-                    'hyper_v_partitions' = @{
-                        'command' = "CheckCounter -a Counter=`"\\Hyper-V Hypervisor\\Partitions`""
-                    };
                     'nova_compute_service_status' = @{
```

Comment:
It is. Just few lines below it's the declaration for it --> [here](https://github.com/ionutbalutoiu/nova-hyperv-charm/blob/master/hooks/local-monitors-relation-changed.ps1#L45-L47)